packages/mds-db/migration.ts

```diff
@@ -166,7 +166,7 @@ async function alterAuditEventsColumnsMigration(exec: SqlExecuterFunction) {
 
 async function alterPreviousGeographiesColumnMigration(exec: SqlExecuterFunction) {
   await exec(
-    `ALTER TABLE ${schema.TABLE.geography_metadata} RENAME COLUMN previous_geographies ${schema.COLUMN_TYPE.prev_geographies}`
+    `ALTER TABLE ${schema.TABLE.geographies} RENAME COLUMN previous_geographies ${schema.COLUMN.prev_geographies}`
```

Comment:
WHOOPS

Reply:
That is very unfortunate and concerning that tests passed.

Reply:
Tests would pass because this code probably never runs as part of the test since the db gets initialized and all tables get created according to the new schema. We probably should think about how we can create specific tests to execute these migrations against.